src/qt/modaloverlay.cpp

```diff
@@ -29,6 +35,15 @@ userClosed(false)
 
     blockProcessTime.clear();
     setVisible(false);
+    bool enableWallet = false;
+#ifdef ENABLE_WALLET
+    enableWallet = WalletModel::isWalletEnabled();
+#endif
+    if (!enableWallet)
+    {
+        ui->infoText->setVisible(false);
+        ui->infoTextStrong->setText("Bitcoin Core is not fully synced yet.");
```

Comment:
This eventually should be written in positive form ("is syncing" instead of "not filly synced")... could also a bit more explanatory